packages/sdk/chainDeploy/helpers/liquidators/registry.ts

```diff
@@ -10,22 +10,33 @@ export const configureLiquidatorsRegistry = async ({
 }: LiquidatorsRegistryConfigFnParams): Promise<void> => {
   const { deployer } = await getNamedAccounts();
 
-  const strategies: string[] = [];
-  const inputTokens: string[] = [];
-  const outputTokens: string[] = [];
   const lr = await ethers.getContract("LiquidatorsRegistry");
   const liquidatorsRegistry = (await ethers.getContractAt(
     "ILiquidatorsRegistry",
     lr.address,
     deployer
   )) as ILiquidatorsRegistry;
 
+  let strategies: string[] = [];
+  let inputTokens: string[] = [];
+  let outputTokens: string[] = [];
+  const strategyMapping: { [inputToken: string]: { [outputToken: string]: string } } = {};
+
+  [strategies, inputTokens, outputTokens] = await liquidatorsRegistry.callStatic.getAllPairsStrategies();
+  inputTokens.map((inputToken, index) => {
+    strategyMapping[inputToken] = { [outputTokens[index]]: strategies[index] };
+  });
+
   for (const inputToken in chainIdToConfig[chainId].redemptionStrategies) {
     const [redemptionStrategyType, outputToken] = chainIdToConfig[chainId].redemptionStrategies[inputToken];
     const redemptionStrategy = await ethers.getContract(redemptionStrategyType, deployer);
-    strategies.push(redemptionStrategy.address);
-    inputTokens.push(inputToken);
-    outputTokens.push(outputToken);
+
+    const existingStrategy = strategyMapping[inputToken][outputToken] ?? null;
+    if (existingStrategy != redemptionStrategy.address || existingStrategy == null) {
+      strategies.push(redemptionStrategy.address);
+      inputTokens.push(inputToken);
+      outputTokens.push(outputToken);
+    }
```

Comment:
this will result in configuring only the strategies that are different, but all that are not different will be removed from the on-chain config...